Cargo.toml

```diff
@@ -20,3 +22,5 @@ thiserror = "1.0.38"
 [dev-dependencies]
 hex-literal = "0.3.4"
 hex = "0.3.1"
+rand = "0.7.0"
+rand_core = "0.6.4"
```

Comment:
We need both of these because seeding in ed25519-dalek depends on an older version of `rand` and its corresponding trait (`CryptoRng` iirc).